src/Microsoft.DotNet.CodeFormatting/Resources.resx

```diff
@@ -126,6 +126,15 @@
   <data name="ExplicitThisFixer_Title" xml:space="preserve">
     <value>Remove 'this' qualifier</value>
   </data>
+  <data name="OptimizeNamespaceImportsAnalyzer_MessageFormat" xml:space="preserve">
```

Comment:
Suggest calling the analyzer itself `UnusedNamespaceDirectivesAnalyzer`. Just my 2c.